networks/src/calibnet/mod.rs

```diff
@@ -5,6 +5,8 @@ use super::{drand::DRAND_MAINNET, DrandPoint, Height, HeightInfo};
 
 /// Default genesis car file bytes.
 pub const DEFAULT_GENESIS: &[u8] = include_bytes!("genesis.car");
+/// Genesis CID
+pub const GENESIS_CID: &str = "bafy2bzacecz3trtejxtzix4f4eebs7dekm6snfsmvffiqz2rfx7iwgsgtieq4";
```

Comment:
Beware of a potential `GENESIS_CID` change for calibnet due to the reset of the chain.

Reply:
updated calibnet `GENESIS_CID` using new genesis file